.github/workflows/publish-libs.yaml

```diff
@@ -19,3 +19,4 @@ jobs:
           CHARMCRAFT_AUTH: "${{ secrets.CHARMCRAFT_AUTH }}"
         run: |
           charmcraft publish-lib charms.kubernetes_charm_libraries.v0.multus
+          charmcraft publish-lib charms.kubernetes_charm_libraries.v0.hugepages_volumes_patch
```

Comment:
Now that we have two libs here, I think we should start checking which of them has changed and publish selectively. 
[Here's an example](https://github.com/canonical/sdcore-upf-operator/blob/main/.github/workflows/main.yaml)